infrastructure/lambda/private-api.yaml

```diff
@@ -126,6 +126,49 @@ paths:
           Fn::Sub: arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/arn:${AWS::Partition}:lambda:${AWS::Region}:${AWS::AccountId}:function:${CommonStackName}-AuthorizationFunction/invocations
         passthroughBehavior: "when_no_match"
 
+  /person-info:
```

Comment:
Are we changing state? This could potentially be a get request

Reply:
I had originally thought the session_id would be in the body and therefore needed as a post, but seeing as we're passing that as a header you're right this can be a get